go/vt/vtgate/planbuilder/testdata/show_cases.txt

```diff
@@ -243,3 +243,20 @@
     "OperatorType": "Rows"
   }
 }
+
```

Comment:
shouldn't we have more examples of the new `show` plans?

Reply:
added